include/mata/nfa/delta.hh

```diff
@@ -71,7 +72,7 @@ public:
 
     size_t count(State s) const { return targets.count(s); }
     bool empty() const { return targets.empty(); }
-    size_t size() const { return targets.size(); }
+    size_t num_of_targets() const { return targets.size(); }
```

Comment:
Why not size?

Reply:
We agreed that it is confusing having to constantly wonder about what `size()` on this and that particular class means. If it is not a simple class like a set (`OrdVector()`, `SparseSet`, etc.), we should not use `size()`. Do you think `SymbolPost::size()`, that is, `if (3 != symbol_post.size())`, for example, is immediately clear? I would prefer to see that we are talking about the number of targets in that symbol post.

Reply:
ok, keep num_of_targets